MoneyTransferWorker/BankingService.cs

```diff
@@ -0,0 +1,88 @@
+namespace MoneyTransferProject;
+
+public class Account
+{
+    public string AccountNumber { get; private set; }
+    public int Balance { get; set; }
+
+    public Account(string accountNumber, int balance)
+    {
+        AccountNumber = accountNumber;
+        Balance = balance;
+    }
+}
+
+public class Bank
+{
+    private List<Account> _accounts;
+
+    public Bank(List<Account> accounts)
+    {
+        _accounts = accounts ?? new List<Account>();
+    }
+
+    public Account FindAccount(string accountNumber)
```

Comment:
Arguably not that valuable for a helper, caller can just look through the account list themselves